index.html

```diff
@@ -63,6 +65,55 @@ <h1>
 
     <div class="container">
 
+        <section class="main">
+            <div class="container">
+                <div class="row">
+                    <div class="col-lg-3 main_one">
+                        <div class="main_categories">
+                            <div class="main_categories_all">
+                                <i class ="fa fa-bars"></i>
+                                <span> All categories</span>
+                            </div>
+                            <ul>
+                                <li><a href=#></a></li>
+                                <li><a href=#></a></li>
+                                <li><a href=#></a></li>
+                                <li><a href=#></a></li>
+                                <li><a href=#></a></li>
+                                <li><a href=#></a></li>
+                                <li><a href=#></a></li>
+                                <li><a href=#></a></li>
+                                <li><a href=#></a></li>
+                                <li><a href=#></a></li>
+                                <li><a href=#></a></li>
+                                <li><a href=#></a></li>
+                                <li><a href=#></a></li>
+                                <li><a href=#></a></li>
+                                <li><a href=#></a></li>
+                                <li><a href=#></a></li>
+                                <li><a href=#></a></li>
+                                <li><a href=#></a></li>
+                                
+                            </ul>
+                        </div>
+                        
+
+                    </div>
+                    <div class="col-lg-9 main_two" style="background-size: 100%; background-image:url('images/bag.jpg');">
+                        <div class="main__item" >
+                            <div class="main__text">
+                                <span>FRUIT FRESH</span>
```

Comment:
I think you don;t need to hardcode this product in here, because you'll got products from api